_docs/platforms/kubernetes/install/1.10.md

```diff
@@ -7,7 +7,7 @@ redirect_from: /docs/install/schedulers/kubernetes
 redirect_from: /docs/install/kubernetes
 ---
 
-# Kubernetes 1.10+
+# Kubernetes 1.10 and 1.11
```

Comment:
Maybe add a link to the system configurations page after this and a note that linux modules must be installed before following the following instructions.

Reply:
I added a note warns people to follow the prerequisites before installing.

Reply:
done